autobump.txt

```diff
@@ -0,0 +1,1320 @@
+abcmidi
```

Comment:
Can you put this file under `.github/workflows`? Thanks!

Reply:
I think I tried but this breaks GitHub: GitHub Actions will try to load it and complain about not finding an autobump module

Reply:
@MikeMcQuaid we have have similar files / folders at the root already:
```
formula_renames.json,  pypi_formula_mappings.json, style_exceptions, synced_versions_formulae.json, tap_migrations.json
```

Reply:
Yeh, I'd rather we stopped adding more stuff there, particularly if it's actions-specific.

It may be putting it in `.github/` instead of `.github/workflows` will work?

Reply:
Moving to `.github/workflows` worked.